solidity/test/LiquidityProtection.js

```diff
@@ -2201,35 +2201,35 @@ describe('LiquidityProtection', () => {
                     it('should update the last removal check point when transferring liquidity', async () => {
                         expect(await checkpointStore.checkpoint.call(recipient)).to.be.bignumber.equal(new BN(0));
 
-                        await liquidityProtection.transferLiquidity(protectionId, newOwner, {
```

Comment:
Need to cast `lockedCount` from `BN` to `Number` in line 288 or in line 289.

Reply:
Please note that this has nothing to do with the changes in this PR nor in the changes/additions we do when touching old code (such as style, formatting, documentation, refactoring, etc.), so this by no means should be a "requested changed" for this PR.

Reply:
I don't see a conceptual difference between fixing minor issues such as style, formatting, documentation, refactoring, etc., and fixing the minor issue mentioned above. The alternative of opening a PR just for this one minor fix is IMO worse than following the 'style, formatting, documentation, refactoring' artificial restriction.